stable/concourse/templates/worker-statefulset.yaml

```diff
@@ -90,6 +71,18 @@ spec:
               readOnly: true
             - name: concourse-work-dir
               mountPath: /concourse-work-dir
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
+            podAffinityTerm:
+              labelSelector:
+                matchExpressions:
```

Comment:
do you want to swtich to `matchLabels` instead of `labelSelector` the former is easier to use. 
```
labelSelector:
              matchLabels:
                app:  {{ template "fullname" . }}
                release: {{ .Release.Name | quote }}
```